content/_index.md

```diff
@@ -7,7 +7,11 @@ type: docs
 
 ## What is OSM?
 
-Open Service Mesh (OSM) runs an Envoy based control plane on Kubernetes, can be configured with SMI APIs, and works by injecting an Envoy proxy as a sidecar container next to each instance of your application. The proxy contains and executes rules around access control policies, implements routing configuration, and captures metrics. The control plane continually configures proxies to ensure policies and routing rules are up to date and ensures proxies are healthy.
+Open Service Mesh (OSM) is a simple, complete, and standalone [service mesh](https://en.wikipedia.org/wiki/Service_mesh).
+
+OSM provides a fully featured control plane. It leverages an architecture based on [Envoy](https://www.envoyproxy.io/) reverse-proxy sidecar and works by injecting an Envoy proxy as a sidecar container next to each instance of your application.
+
+OSM can be configured with SMI APIs; it relies on [SMI Spec](https://smi-spec.io/) to reference services that will participate in the service mesh. OSM ships out-of-the-box with all necessary components to deploy a complete service mesh spanning multiple compute platforms.
```

Comment:
SMI is not a hard depenency. Permissive mode is an option to not use SMI APIs to route traffic within the mesh. A mention regarding `SMI or Permissive made to configure traffic routing` would clarify this.

Reply:
Revised this section.